src/diepvries/hub.py

```diff
@@ -81,16 +81,24 @@ def sql_placeholders(self) -> Dict[str, str]:
         """
         hashkey = next(hashkey for hashkey in self.fields_by_role[FieldRole.HASHKEY])
 
-        fields = ", ".join(format_fields_for_select(fields=self.fields))
+        target_fields = ", ".join(format_fields_for_select(fields=self.fields))
         staging_fields = ", ".join(
             format_fields_for_select(fields=self.fields, table_alias="staging")
         )
+        source_fields = [
+            field
+            for field in self.fields
+            if field.role != FieldRole.HASHKEY
+            and field.name != METADATA_FIELDS["record_source"]
+        ]
+        source_fields_sql = ", ".join(format_fields_for_select(fields=source_fields))
```

Comment:
I'd use the same name pattern for source and target, i.e. here we have `target_fields` and `source_fields_sql` being the same type of things, but one is suffixed with `_sql`

Reply:
In this case I separated because one variable is the list of fields, and the other variable is the fields as string, prepared for SQL. Using only one variable would look like this (which I'm also ok with):
```
", ".join(
            format_fields_for_select(
                fields=[
                    field
                    for field in self.fields
                    if field.role != FieldRole.HASHKEY
                    and field.name != METADATA_FIELDS["record_source"]
                ]
            )
        )
```

Reply:
I'd keep 2 variables but rename them so that the same pattern is applied for both source and target

Reply:
In this case I could actually just discard the `format_fields_for_select` function and just use:
```
source_fields = ", ".join(
                fields=[
                    field.name
                    for field in self.fields
                    if field.role != FieldRole.HASHKEY
                    and field.name != METADATA_FIELDS["record_source"]
                ]
        )
```

This whole `format_fields_for_select` should actually be reconsidered, only used when we need the field to be aliased eventually.

Reply:
Done anyway!